modules/helper.rb

```diff
@@ -113,6 +121,21 @@ def self.is_verified?(event, other_user = nil)
       return is_xxx_role?(event, 'vfd', 'Verified', true, user)
     end
 
+    def self.get_roles(event)
+      roles = []
```

Comment:
I'd highly advise against doing this in the first place. It's slow enough to check by events which tbh I should've never done in the first place. Perhaps a better idea would be to cache the roles somehow for a user; code lookup is slow enough as it is. I was mulling over ideas on how to improve the actual code part earlier, perhaps I should start thinking on badges as well.